addons/binding/org.openhab.binding.nest/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,319 @@
+        <state readOnly="true" />
+    </channel-type>
+
+    <channel-type id="PeakPeriodEndTime" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Peak Period End Time</label>
+        <description>Peak period end for the Rush Hour Rewards program</description>
+        <state readOnly="true" />
+    </channel-type>
+
+    <channel-type id="EtaBegin" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>ETA</label>
+        <description>
+            Estimated time of arrival at home, will setup the heat to turn on and be warm
+            by the time you arrive.
+        </description>
+    </channel-type>
+
+    <channel-type id="RushHourRewardsEnrollment">
+        <item-type>Switch</item-type>
+        <label>Rush Hour Rewards</label>
+        <description>If rush hour rewards system is enabled or not.</description>
+        <state readOnly="true" />
+    </channel-type>
+
+    <!-- Camera -->
+    <channel-type id="AudioInputEnabled" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Audio Input Enabled</label>
+        <description>If the audio input is enabled for this camera.</description>
+    </channel-type>
+
+    <channel-type id="VideoHistoryEnabled" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Video History Enabled</label>
+        <description>If the video history is enabled for this camera.</description>
+    </channel-type>
+
+    <channel-type id="PublicShareEnabled" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Public Share Enabled</label>
+        <description>If the public sharing of this camera is enabled.</description>
+    </channel-type>
+
+    <channel-type id="Streaming">
+        <item-type>Switch</item-type>
+        <label>Streaming</label>
+        <description>If the camera is currently streaming.</description>
+    </channel-type>
+
+    <channel-type id="WebUrl">
+        <item-type>String</item-type>
+        <label>Web Url</label>
+        <description>The web url for the camera, allows you to see the camera in a web page.</description>
+    </channel-type>
+
+    <channel-type id="PublicShareUrl">
+        <item-type>String</item-type>
+        <label>Public Share Url</label>
+        <description>The url for the camera which is open for a public share.</description>
+    </channel-type>
+
+    <channel-type id="SnapshotUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>Snapshot Url</label>
+        <description>The url for the camera showing a snapshot of the camera.</description>
+    </channel-type>
+
+    <channel-type id="AppUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>App Url</label>
+        <description>The app url for the camera, allows you to see the camera in an app.</description>
+    </channel-type>
+
+    <!-- smoke detector -->
+    <!-- TODO could / should we use a real color item instead -->
```

Comment:
@wborn wdyt?

Reply:
They messed up their API by using colors for communicating the severity of problems. Instead they should have come up with something like warning, error, fatal here. 
Color blind people will not appreciate these colors and want others. Turning it into real colors makes using this state even harder.

Reply:
I ll remove the todo